v0/pipeline/stages/custom/execution-wrapper-config.yaml

```diff
@@ -76,6 +76,7 @@ properties:
     - $ref: ../../steps/common/aws-cdk-rollback-step-node.yaml
     - $ref: ../../steps/cd/download-aws-s3-step-node.yaml
     - $ref: ../../steps/cd/download-harness-store-step-node.yaml
+    - $ref: ../../steps/common/dbops-liquibase-command-step-node.yaml
```

Comment:
why we are adding only to custom?
I think apply and rollback are added to cd and approval stages also.

Reply:
updated the step node to both cd and approval stages.